### PR TITLE
feat: add resources for lesson 6 & 7 JS basics (lesson-6)

### PR DESCRIPTION
**Pull request by jozsef-gal-epam**

### CODE REVIEW

File: client/lesson-6/code/3a-async-timer.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Example #3 - Asnych call with setTimeout</title>
```
Comment (veresdavid) [inline client/lesson-6/code/3a-async-timer.html]: Typo: `Asnych` instead of `Async`

File: client/lesson-6/code/3b-async-timer.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Example #3 - Asnych call with setTimeout</title>
```
Comment (veresdavid) [inline client/lesson-6/code/3b-async-timer.html]: Same typo here

File: client/lesson-6/code/3c-async-timer-with-promise.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Example #3 - Asnych call with setTimeout</title>
```
Comment (veresdavid) [inline client/lesson-6/code/3c-async-timer-with-promise.html]: Same typo here

**Review by veresdavid — CHANGES_REQUESTED** [review]

File: client/lesson-6/code/3a-async-timer.html (hunk 1 above)
Comment (jozsef-gal-epam) [reply client/lesson-6/code/3a-async-timer.html]: Thanks - fixed
Copy-paste rulez :)

**Review by veresdavid — APPROVED** [review]

**Pull request merged**